### DISCUSSION TITLE
Add on Superset a Rich Text Chart Or BigText Chart

### DISCUSSION THREAD

**Discussion by zsellami** [apache/superset#18509] [Ideas]
As a user, I want to show big text information obtained from a dataset. For instance by adapting The Big Number chart to allow to show text instead of number.

The Text chart can usefull to show important informations on dashboard. For instance, the name of the best employee, the name of the first seller, etc.

Enable HTML script will also help to add color and size in this kind of chart.
Here some sample of what teh chart can be:

![Screenshot_20210923_140330](https://user-images.githubusercontent.com/22955406/134503477-36be5ba9-812f-4186-a47c-07c5841b0abb.png)

**Comment by kamalkeshavani-aiinside** [comment]
related to https://github.com/apache/superset/issues/12516
